terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1830,9 +1830,10 @@ def qq(*words):
                                 'Requires=docker.service gitlab-dind.service',
                                 '[Service]',
                                 # We explicitly configure Docker (see /etc/docker/daemon.json) to log to
-                                # journald, so we don't need systemd to capture process output.
-                                'StandardOutput=null',
-                                'StandardError=null',
+                                # journald, which works for `docker run` but not `docker exec`, so for
+                                # these logs we let systemd control where the process output gets logged.
+                                'StandardOutput=journal',
+                                'StandardError=inherit',
```

Comment:
Why are they different?

Reply:
The default for `StandardError` is `inherit`

https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#StandardError=

> The available options are identical to those of StandardOutput=, with some exceptions: if set to inherit the file descriptor used for standard output is duplicated for standard error, while fd:name will use a default file descriptor name of "stderr".

Instead of explicitly specifying these directives with the same values as their default, my fixup commit simply removes them. I've kept the comment for these directives however and updated it accordingly.